src/app/public/modules/action-button/action-button-container.component.scss

```diff
@@ -1,40 +1,55 @@
 @import "~@skyux/theme/scss/mixins";
 @import "~@skyux/theme/scss/_compat/mixins";
 
-@media (min-width: $sky-screen-sm-min) {
-  .sky-action-button-container {
-    display: flex;
-    flex-wrap: wrap;
-    align-items: stretch;
+.sky-action-button-container {
+  // Note: this has to go on the element as the flex stretch will consume the margin if not.
+  ::ng-deep sky-action-button {
+    margin: $sky-margin 0;
   }
 }
 
-@media (max-width: $sky-screen-xs-max) {
+@include sky-host-responsive-container-xs-min() {
   .sky-action-button-container {
+    display: block;
```

Comment:
Why are we shying away from flexbox here? Couldn't we still achieve the same result with `flex-direction: column`?

Reply:
Ah, you're just replicating what it did before. I'm not sold on this solution, but it shouldn't hold up your PR. Disregard.